### PR TITLE
Add @Akuli as maintainer

### PR DESCRIPTION
**Pull request by srittau**

### CODE REVIEW

**Review by JelleZijlstra — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -78,6 +78,7 @@ Also, do not squash your commits after you have submitted a pull request, as thi
 erases context during review. We will squash commits when the pull request is merged.
 
 At present the core developers are (alphabetically):
+* Akuli (@Akuli)
```
[the remark this answers is not in the document]
Comment (Akuli) [reply CONTRIBUTING.md:81]: My real name is somewhat visible in the email address that I commit with, and I'm fine with this either way. I might later make a separate PR that changes this to "maintainer".

**Review by Akuli — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (Akuli) [inline CONTRIBUTING.md:81]: It seems that real name was slightly more preferable, after all:

```suggestion
* Aku Viljanen (@Akuli)
```

**Review by Akuli — CHANGES_REQUESTED** [review]

**Comment by srittau** [comment]
@Akuli Would you like to do the honors of merging?

**Comment by Akuli** [comment]
Oops, merged before tests passed, although it doesn't matter much with this PR. I'll have to get used to slowly running tests.

**Pull request merged**